api/turing/service/router_deployment_service.go

```diff
@@ -254,7 +255,7 @@ func (ds *deploymentService) UndeployRouterVersion(
 
 	// Delete secret
 	eventsCh.Write(models.NewInfoEvent(models.EventStageDeletingDependencies, "deleting secrets"))
-	secret := ds.svcBuilder.NewSecret(routerVersion, project, "", "", "")
+	secret := ds.svcBuilder.NewSecret(routerVersion, project, ds.environmentType, "", "", "")
```

Comment:
Similar comment as above: why are we sending in the envType here ? Looks like the methods in `api/turing/cluster/servicebuilder/service_builder.go` no longer need it.